src/extension.ts

```diff
@@ -2,6 +2,11 @@
 // Import the module and reference it with the alias vscode in your code below
 import * as vscode from "vscode";
 import { injectCompletionModel } from "./helpers/injectCompletionModel";
+import { CompletionModelResponse } from "./CompletionModel/CompletionModel";
+
+const codeReviewInstruction =
```

Comment:
This looks good for now, but maybe in the future we should import all these from the same file. They're also true constants, so should probably be named `CODE_REVIEW_INSTRUCTION` and `CODE_REVIEW_PROGRESS_TITLE`.

Reply:
I agree. I wasn't sure the typescript conventions on constants and meant to look into that before pushing. Thanks for pointing that out!